tests/unit/storage_v3.cpp

```diff
@@ -12,10 +12,2575 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <limits>
+
+#include "storage/v3/property_value.hpp"
 #include "storage/v3/storage.hpp"
+#include "storage/v3/vertex_accessor.hpp"
+#include "storage_v3_test_utils.hpp"
+
+using testing::UnorderedElementsAre;
+
+namespace memgraph::storage::v3::tests {
+
+// NOLINTNEXTLINE(hicpp-special-member-functions)
+TEST(StorageV3, Commit) {
+  Storage store;
```

Comment:
You have storage in every test, and there are a lot of them, maybe move it to fixture jsut to avoid repetition

Reply:
Plus another question are all of these tests from v2, and is there something new here

Reply:
I think they are 1-to-1 copies of v2 tests. I think there was no issue with compilation etc, so I only "shortened" the source by putting it into the `memgraph::storage::v3::tests` namespace and remove the extraneous prefixes.